x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -1227,6 +1227,29 @@ vals:l
 183
 ;
 
+emptyProjectInStatWithEval
+FROM employees
+| STATS c = COUNT(salary)
+| EVAL x = 3.14
+| DROP c
+;
+
+x:d
+3.14
+;
+
+emptyProjectInStatWithGroupAndEval
+FROM employees
+| STATS c = COUNT(salary) BY emp_no
```

Comment:
Please add another test, one without any agg (just grouping - STATS by emp_no), another one with multiple groups and another one with a diff agg than count (e.g STATS MIN(salary) by languages).